extension-push/src/push_android.cpp

```diff
@@ -307,7 +307,8 @@ static int Push_GetScheduled(lua_State* L)
             return 1;
         }
     }
-    return 0;
+    lua_pushnil(L);
```

Comment:
We could obviously get the stack top and compare instead of using DM_LUA_STACK_CHECK but DM_LUA_STACK_CHECK is much more convenient.